### DISCUSSION TITLE
ROCm and Tensorflow Installation GPU not used

### DISCUSSION THREAD

**Discussion by GMellow** [ROCm/ROCm#2443] [Q&A]
Hi. I have successfully installed ROCm 5.6.  I'm a bit (a lot) confused by what are the steps I should take to make sure I can use Tensorflow (2.10) with Python (3.10.12) on my RX 6900 XT GPU. There seems to be a variety of outdated, and/or conflicting information. Do I need to use the TensorFlow GPU version?  Some other version? Some other setting I'm missing? The code runs perfectly after the ROCm 5.6 installation but does not seem to be using the GPU (CPU only). Sorry if this question is answered elsewhere....I tried to find the answer on my own but kept getting conflicting paths ahead....

**Comment by briansp2020** [comment]
Did you try ? 
>pip install tensorflow-rocm
